src/Annotation/Parser/OptionParser.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+namespace Framework3\Annotation\Parser;
+
+use Framework3\Exception\AppException;
+
+class OptionParser
+{
+    /**
+     * This option is used when parsing annotation options.
+     * It returns parsed data as an array.
+     */
+    private const RETURN_TYPE_OBJECT = 1;
+
+    /**
+     * This option is used when parsing annotation options.
+     * It returns parsed data as an array of OptionData objects.
+     */
+    private const RETURN_TYPE_ARRAY  = 2;
```

Comment:
Codacy found an issue: [Equals sign not aligned correctly; expected 1 space but found 2 spaces](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6283010)